app/src/main/kotlin/com/metrolist/music/widget/MusicWidgetReceiver.kt

```
@@ -42,43 +41,124 @@ class MusicWidgetReceiver : AppWidgetProvider() {
                 }
             }
         }
     }
 
+    private fun triggerServiceUpdate(context: Context) {
+        val intent = Intent(context, MusicService::class.java).apply {
+            action = ACTION_UPDATE_WIDGET
+        }
+        try {
+            context.startService(intent)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)